NitroxModel/Discovery/InstallationFinders/ConfigGameFinder.cs

```diff
@@ -1,30 +1,43 @@
-﻿using System.Collections.Generic;
+using System.Collections.Generic;
 using System.IO;
+using NitroxModel.Discovery.Models;
 using NitroxModel.Helper;
 
-namespace NitroxModel.Discovery.InstallationFinders
+namespace NitroxModel.Discovery.InstallationFinders;
```

Comment:
The steps in `FindGame` are more or less the same across all game finders. Maybe we can have a central method (for checking for null, Managed exists, game.exe exists) somewhere.